src/ToggleButton/index.jsx

```diff
@@ -0,0 +1,182 @@
+import React               from 'react';
+import PropTypes           from 'prop-types';
+
+import { buildClassName }  from '../utils';
+import styles              from './toggleButton.css';
+import Icon                from '../Icon/';
+
+const ToggleButton = ( {
+    children,
+    className,
+    cssMap,
+    iconPosition,
+    iconSize,
+    iconType,
+    id,
```

Comment:
If no `id` is supplied as a prop, we should generate one. We have `generateId` util to do this:

```
...
iconType,
id = generateId( 'ToggleButton' ),
isDisabled,
...
```

(You'll need to import `generateId` from `../utils` at the top.)